Tools/Cli-LoRa-Device-Provisioning/CommandLineOptions/AddGatewayOption.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tools.CLI.Options
+{
+    using System;
+    using CommandLine;
+
+    [Verb("add-gateway", HelpText = "Add a new gateway device to IoT Hub.")]
+    public class AddGatewayOption
```

Comment:
We have been avoiding the term gateway lately, what do you think about naming this "Add iot-edge"?

Reply:
Sure, I wasn't aware.
What is the name we use to describe to users? Iot edge device? Network server device?

Reply:
As discussed tags will be:

- lora_device_type: an array containing
  - network_server
  - basics_station
  - leaf
  - concentrator

Verb will be:
add-gateway